recipes/libdispatch/all/patches/0002-Fix-Windows-compatibility.patch

```diff
@@ -0,0 +1,24 @@
+From 92557823c507780cf3fad2bfe83fedfa36fe844a Mon Sep 17 00:00:00 2001
```

Comment:
has this commit been merged upstream? I'm unable to find a PR or commit in the upstream repo (the other patch has https://github.com/swiftlang/swift-corelibs-libdispatch/pull/895, but it does not cover this change), and this bit of logic has been removed in the most recent head branch